calico/networking/configuring/qos-controls.mdx

```diff
@@ -0,0 +1,91 @@
+---
+description: Configure QoS (Quality of Service) Controls to limit ingress and/or egress bandwidth, packet rate and number of connections of Calico workloads.
+---
+
+# Configure QoS Controls
+
+## Big picture
+
+Configure QoS (Quality of Service) Controls to limit ingress and/or egress bandwidth, packet rate and number of connections used by Calico workloads in order to prevent them from overusing or overloading network resources.
+
+## Value
+
+With QoS controls, $[prodname] enforces network resource limits (bandwidth, packet rate, connections) for Kubernetes pods and OpenStack VMs, ensuring fair resource allocation and preventing performance degradation for other workloads.
+
+## Concepts
+
+### Quality of Service Controls
+
+**Quality of Service**, or **QoS** for short, in the field of computer networking, refers to traffic prioritization and resource reservation control mechanisms. It may either mean guaranteeing minimum levels of networking resources, or limiting the maximum levels of usage of networking resources in order to prevent it from being overloaded by a small number of actors. For practical purposes, the latter will be considered as $[prodname] implements QoS Controls in this manner.
```

Comment:
Title says QoS, then Big Picture says `Configure QoS (Quality of Service)` and here you define QoS again in reverse :man_shrugging: Imo QoS is well established - if not, using it the Title is too early. But once you define it, you can just keep using QoS

```suggestion
**Quality of Service (QoS)** for short, in the field of computer networking, refers to traffic prioritization and resource reservation control mechanisms. It may either mean guaranteeing minimum levels of networking resources, or limiting the maximum levels of usage of networking resources in order to prevent it from being overloaded by a small number of actors. For practical purposes, the latter will be considered as $[prodname] implements QoS Controls in this manner.
```